src/encoded/types/antibody_lot.py

```diff
@@ -158,151 +162,166 @@ def lot_reviews(characterizations, targets, request):
         target_organisms[organism] = target['@id']
     target_organisms['all'] = tmp
 
+    # The default if no characterizations have been submitted
+    base_review = {
+        'biosample_term_name': 'any cell type or tissue',
+        'biosample_term_id': 'NTR:99999999',
+        'organisms': sorted(target_organisms['all']),
+        'targets': sorted(targets),
+        'status': 'characterized to standards with exemption'
+        if is_control else ab_states[(None, None)],
+        'detail': 'IgG does not require further characterization.'
+        if is_control else ab_state_details[(None, None)]
+    }
+
     if not characterizations:
         # If there are no characterizations, then default to awaiting characterization.
-        return [{
-            'biosample_term_name': 'any cell type or tissues',
-            'biosample_term_id': 'NTR:99999999',
-            'organisms': sorted(target_organisms['all']),
-            'targets': sorted(targets),  # Copy to prevent modification of original data
-            'status': 'characterized to standards with exemption' if is_control else 'awaiting characterization',
-            'detail': 'IgG does not require further characterization.' if is_control else 'No characterizations submitted for this antibody lot yet.'
-        }]
+        return [base_review]
 
     review_targets = set()
-    primary_chars = []
-    secondary_chars = []
+    char_organisms = dict()
```

Comment:
I think the usual is to declare:
`char_organisms = {}`
and
`primary_chars = []`